application/models/User.php

```diff
@@ -694,17 +694,37 @@ public function getManagementColums()
             ],
         ];
 
-        if (Permission::model()->hasGlobalPermission('superadmin', 'read')) {
+        // NOTE: Super Administrators with just the "read" flag also have these flags
+        $permission_read_users      = Permission::model()->hasGlobalPermission('users', 'read');
+        $permission_read_usergroups = Permission::model()->hasGlobalPermission('usergroups', 'read');
+        $permission_read_surveys    = Permission::model()->hasGlobalPermission('surveys', 'read');
+
+        // Number of Surveys
+        // This info is already guessable by people able to list all Surveys
+        if ($permission_read_surveys) {
             $cols[] = array(
                 "name" => 'surveysCreated',
                 "header" => gT("No of surveys"),
                 'filter' => false
             );
+        }
+
+        // Usergroups Names
+        // This info is safe to be shown to who can read all Users and Groups.
+        // TODO: When there will be a more robust Group permissions system,
+        //       this column could be enabled by default, since each Group would
+        //       be checked individually.
+        if ($permission_read_users && $permission_read_usergroups) {
```

Comment:
I don't have a strong opinion on this, but,

Probably this simply should be "if I can see all the groups".